# Rethrow exception

Re-throws the exception that is currently being handled by a catch block.